app/config-utils.js

```diff
@@ -206,6 +206,10 @@ export function fixRelativeUrls(url, config) {
   let urlBase = new URL(url || here()).pathname;
   for (const [k, v] of Object.entries(config)) {
     if (k.endsWith('Url') && v.startsWith('./')) {
+      if (k === 'themesUrl') {
```

Comment:
This was kind of a fast hack, but maybe we should hoist this to `UNPREFIXED_PATHS`?

Reply:
Yes it's probably cleaner in the long run.

Reply:
Like i said below, i'll hit it on #233

Reply:
:+1: